modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -1,10 +1,10 @@
-= Create and Publish a RAML API Fragment in the API editor in API Designer
+= Create and Publish a RAML API Fragment in the code editor in API Designer
```

Comment:
Should be "Code Editor" in a title

Reply:
Thanks. I did a project-wide correction and missed that.